lti/views/about.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+
+from pyramid.view import view_config
+
+from pyramid.response import FileResponse
+
+
+@view_config(route_name='about')
+def about(request):
+    path = '.'
+    file_name = 'about.html'
```

Comment:
This could just be `path = ./about.html` and then the `'%s/%s'` below wouldn't be needed